### ISSUE TITLE
Firestore test error shows: Ref at provided path 'xxx/yyy' does not have action "add"

### ISSUE THREAD

**Issue by axacheng** [prescottprue/cypress-firebase#7]
I assume it's something missing while i follow along the doc, so i can not add test data to Firestore.
Plus, I checked my Firestore rule that allows write with everyone(bad, but just for testing for now.).
Please try to help us.

Thanks 

```bash
CypressError: cy.exec('$(npm bin)/firebase-extra firestore add axatest/HUD9SxZHjmrBkGFo5Fl6') failed because the command exited with a non-zero code.

Pass {failOnNonZeroExit: false} to ignore exit code failures.

Information about the failure:
Code: 1

Stderr:
Error running firebase action: Ref at provided path "axatest/HUD9SxZHjmrBkGFo5Fl6" does not have action "add"
```

**Comment by prescottprue** [comment]
You are trying to call `add` on a document instead of on a collection. `add` creates a new document with a unique key, so providing the key makes.

**Comment by axacheng** [comment]
Hi Scott,

Thanks for your reply. It makes sense to me. I was updated the README example please see my PR.

```
Examples
Basic
cy.callFirestore('add', 'project/test-project', 'fakeProject.json')
```

**Comment by prescottprue** [comment]
Merged and released as part of [`v0.1.1`](https://github.com/prescottprue/cypress-firebase/releases/tag/v0.1.1), thanks for the PR.
